BrunoSilvaATB/coatrack
# Enhancement/#67 non initialized gateway should respond with internal server error to service call

How to test this feature:
* Setup Coatrack.
* Create a gateway via tutorial.
* Download the gateway.
* Stop the WebApplication.
* Start the Gateway. It will not be able to fetch ApiKeys from the Web Application as it is down. The Gateway is not initialized with ApiKeys.
* Anyhow, make a service call via Browser.
* A Whitelabel error page should appear showing a 500 internal server error.

closes #67
